src/battle_animation.cpp

```diff
@@ -108,8 +108,10 @@ void BattleAnimation::DrawAt(Bitmap& dst, int x, int y) {
 		}
 
 		SetX(cell.x + x);
+		if (invert) SetX(x - cell.x);
```

Comment:
Minor performance nit: You're calling `SetX()` twice here. `Sprite::SetX()` is currently inlined so right now it doesn't matter, but later if `SetX()` ever changed to be out of line this would be slower. 

Lets make it a little more future proof by computing the new x once and then calling `SetX()`. You could use a ternary operator or create a temporary variable and assign it.